opentracing-mdc-demo/src/main/java/io/opentracing/mdcdemo/MDCActiveSpanHolder.java

```diff
@@ -0,0 +1,66 @@
+package io.opentracing.mdcdemo;
+
+import io.opentracing.ActiveSpanHolder;
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import org.slf4j.MDC;
+
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * MDCActiveSpanHolder illustrates the core ActiveSpanHolder concepts and capabilities to a first approximation. Not
+ * production-quality code.
+ */
+public class MDCActiveSpanHolder extends ActiveSpanHolder {
+    private final ThreadLocal<MDCContinuation> tlsSnapshot = new ThreadLocal<MDCContinuation>();
+
+    class MDCContinuation extends Continuation {
+        private final Map<String, String> mdcContext;
```

Comment:
@sjoerdtalsma this was just a bug introduced by a rebase or something... I am now setting the MDC context during activate(), and also added a "test" (sic) to the MDCDemo.java which works as intended. Good catch, though!